CHANGES.md

```diff
@@ -1,3 +1,26 @@
+## edge-19.7.5
+
+* CLI
+  * Improved the installation report produced by the `linkerd check` command
+    to include the control plane pods' live status
```

Comment:
Maybe "liveness statuses" here, but I'll defer to resident Linkerd Grammatician @scottcarol

Reply:
"Live status" works for me, seems simpler